crates/curp/src/client/mod.rs

```diff
@@ -64,6 +64,7 @@ pub trait ClientApi {
     async fn propose(
         &self,
         cmd: &Self::Cmd,
+        token: Option<&String>, // TODO: Allow external custom interceptors, do not pass token in parameters
```

Comment:
Please submit an issue to track this TODO

Reply:
#657